fintrack/users/models.py

```diff
@@ -0,0 +1,10 @@
+from django.db import models
+from django.contrib.auth.models import AbstractUser
+
+from . user_manager import UserManager
+
+
+class User(AbstractUser):
+    address = models.CharField(max_length=250)
```

Comment:
do we need email and username separate or one should be enough?

Reply:
Username is for login purposes as it is short and handy. where as we are taking email for marketing or reset purposes. if user forgot his username or password, an email could be sent to his registered email to reset it.